internal/notifications/notifications.go

```diff
@@ -0,0 +1,89 @@
+package notifications
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/kafka"
+	"github.com/rs/zerolog"
+)
+
+func init() {
+	GetNotificationClient = getNoopNotificationClient
+}
+
+type client struct{}
+
+func getNotificationClient(ctx context.Context) NotificationClient {
+	zerolog.Ctx(ctx).Debug().Msg("Using kafka notification client")
+	return &client{}
+}
+
+func Initialize(ctx context.Context) {
+	zerolog.Ctx(ctx).Debug().Msg("Initialized kafka notification client")
+	GetNotificationClient = getNotificationClient
+}
+
+func (x *client) SuccessfulLaunch(ctx context.Context, reservationId int64) {
+	logger := zerolog.Ctx(ctx)
+	logger.Debug().Msg("Triggering a successful launch notification")
+	rDao := dao.GetReservationDao(ctx)
+	reservation, err := rDao.GetById(ctx, reservationId)
+	if err != nil {
+		logger.Warn().Err(err).Msg("unable to find reservation by id")
```

Comment:
Nitpick:

* Log messages start with capital letter
* Go errors must be all lowercase (linter enforces it)

I know, if I could ever change this decision, I would use lowercase everywhere.